Misc/NEWS.d/next/Core and Builtins/2022-05-31-16-36-30.gh-issue-93382.Jf6gAj.rst

```diff
@@ -0,0 +1 @@
+Speed up the :c:func:`PyCode_GetCode` function which also improves accessing the :attr:`~types.CodeType.co_code` attribute in Python.
```

Comment:
It would be better to describe what it does, rather than mentioning speed.
Something like "Cache the result of the :c:func:`PyCode_GetCode` function to restore O(1) lookup of the :attr:`~types.CodeType.co_code` attribute`